core/crypto/pedersen_hash.go

```diff
@@ -43,3 +43,16 @@ func Pedersen(a *felt.Felt, b *felt.Felt) (*felt.Felt, error) {
 	}
 	return new(felt.Felt).SetBytes(out[:starkware.FeltSize]), nil
 }
+
+func PedersenHash(elems ...*felt.Felt) (*felt.Felt, error) {
+	switch len(elems) {
+	case 0:
+		return nil, errors.New("number of elems must be more than 0")
+	case 1:
+		return Pedersen(new(felt.Felt).SetZero(), elems[0])
```

Comment:
Are you sure this is always the case? As far as I understand `Pedersen(a,b) != Pedersen(b,a)`. So shouldn't it be left to the caller to decide how they want to call Pedersen when they only want to hash a since value?